pennylane/estimator/templates/subroutines.py

```diff
@@ -1753,6 +1756,9 @@ def resource_decomp(
             :code:`restored = False` and `Berry et al. (2019) <https://arxiv.org/pdf/1902.02134>`_
             (Figure 4) for :code:`restored = True`.
 
+            For :code:`restored = False`, we use the measurement based technique described in
+            `arXiv:1092.02134 <https://arxiv.org/abs/1902.02134>_` to further optimize the cost.
+
```

Comment:
I think these can be merged together. We are already using this paper for the `restored == True` so we should just link the one paper.